utils/html.go

```diff
@@ -120,7 +120,7 @@ func (t *HTMLTemplate) RenderToWriter(w io.Writer) error {
 	return nil
 }
 
-func TranslateAsHtml(t i18n.TranslateFunc, translationID string, args map[string]interface{}) template.HTML {
+func TranslateAsHTML(t i18n.TranslateFunc, translationID string, args map[string]interface{}) template.HTML {
```

Comment:
Can we do this change in a separate PR? I'd like to keep this PR only for moving the service.

Reply:
Because the mattermost-utilities i18n extraction tool has changed to search for `TranslateAsHTML` this is needed to pass the CI's i18n-extract part.

Reply:
Master was already changing that, so... now is fixed anyway.